### ISSUE TITLE
[Feature Request] double click sends app to another workspace

### ISSUE THREAD

**Issue by killown** [micheleg/dash-to-dock#487]
is there a way to implement a double click action which will send the app to another workspace and if possible hide the icon from active workspace? I mean, only hide if it has isolate workspace enabled

**Comment by micheleg** [comment]
I see your point but I'm not sure it makes so much sense to have this action from a click action in the dock. Moreover, a double click will inevitably fist activate whatever the single-click action is and only afterwards will trigger this custom action. Also, to which workspace would the application go? Therefore, it could become a quite ugly effect.

**Comment by killown** [comment]
@micheleg yes, you are right, if the window is minimized it will maximize and send it one workspace down and if the active workspace is the last one then send one workspace up, there are some situations where we don't want to close the window but we want to let the dash more clean, moving them one workspace down is a way to hide it to use later
